boards/recovery/src/data_manager.rs

```diff
@@ -55,65 +51,57 @@ impl DataManager {
             }
         }
         true
     }
     pub fn is_launched(&self) -> bool {
-        match self.sbg_imu1.as_ref() {
-            Some(imu1) => {
-                imu1.accelerometers[1] > ACCEL_MIN
-            },
-            None => {
-                false
-            }
+        match self.imu.0.as_ref() {
+            Some(imu) => imu.accelerometers[1] > ACCEL_MIN,
+            None => false,
         }
     }
     pub fn is_below_main(&self) -> bool {
-        match self.sbg_air.as_ref() {
-            Some(air) => {
-                air.altitude < MAIN_HEIGHT
-            },
-            None => {
-                false
-            }
+        match self.air.as_ref() {
+            Some(air) => air.altitude < MAIN_HEIGHT,
+            None => false,
         }
     }
     pub fn handle_data(&mut self, data: Message) {
         match data.data {
             messages::Data::Sensor(sensor) => match sensor.data {
                 messages::sensor::SensorData::Air(air_data) => {
                     let pressure = air_data.pressure_abs;
-                    self.sbg_air = Some(air_data);
+                    self.air = Some(air_data);
                     if self.historical_pressure.is_full() {
                         self.historical_pressure.dequeue();
                     }
                     self.historical_pressure.enqueue(pressure);
-                },
+                }
                 messages::sensor::SensorData::EkfNav1(nav1_data) => {
-                    self.sbg_nav1 = Some(nav1_data);
-                },
+                    self.ekf_nav.0 = Some(nav1_data);
+                }
                 messages::sensor::SensorData::EkfNav2(nav2_data) => {
-                    self.sbg_nav2 = Some(nav2_data);
-                },
+                    self.ekf_nav.1 = Some(nav2_data);
+                }
                 messages::sensor::SensorData::EkfQuat(quat_data) => {
-                    self.sbg_quat = Some(quat_data);
-                },
+                    self.ekf_quat = Some(quat_data);
+                }
                 messages::sensor::SensorData::GpsVel(gps_vel_data) => {
-                    self.sbg_gps_vel = Some(gps_vel_data);
-                },
+                    self.gps_vel = Some(gps_vel_data);
+                }
                 messages::sensor::SensorData::Imu1(imu1_data) => {
-                    self.sbg_imu1 = Some(imu1_data);
-                },
+                    self.imu.0 = Some(imu1_data);
+                }
                 messages::sensor::SensorData::Imu2(imu2_data) => {
-                    self.sbg_imu2 = Some(imu2_data);
-                },
+                    self.imu.1 = Some(imu2_data);
+                }
                 messages::sensor::SensorData::UtcTime(utc_time_data) => {
-                    self.sbg_utc_time = Some(utc_time_data);
-                },
+                    self.utc_time = Some(utc_time_data);
+                }
             },
             _ => {}
```

Comment:
Clippy: you seem to be trying to use `match` for destructuring a single pattern. Consider using `if let`